### PR TITLE
adds command call logging for all create/update/abandon commands

### LINKED ISSUE #2199: Log stream update and channel create/update parameters
## The Issue
We only added logging for the publish command, not stream update or channel create/update. These are good to have in the logs for debugging. 

## System Configuration

 

- LBRY Daemon version:
- LBRY App version:
- LBRY Installation ID:
- Operating system:

### Acceptance Criteria
1.
2.
3.

### Definition of Done
- [ ]  Tested against acceptance criteria
- [ ] Tested against the assumptions of user story
- [ ] The project builds without errors
- [ ] Unit tests are written and passing
- [ ] Tests on devices/browsers listed in the issue have passed
- [ ] QA performed & issues resolved
- [ ] Refactoring completed
- [ ] Any configuration or build changes documented
- [ ] Documentation updated
- [ ] Peer Code Review performed

### PR DESCRIPTION
**Pull request by osilkin98**
Fixes #2199

### CODE REVIEW

**Comment by lyoshenka** [comment]
@osilkin98 what's the next step here? does it need a review?

**Comment by osilkin98** [comment]
Yes, this is review-ready.

File: lbry/lbry/extras/daemon/Daemon.py [hunk 1]
```diff
@@ -375,6 +375,13 @@ def installation_id(self):
                 install_id_file.write(self._installation_id)
         return self._installation_id
 
+    @staticmethod
+    def log_function_params(fname, **params):
+        params_passed = [f'{k}: {repr(v)}' for k, v in params.items() if v is not None]
+        logstr = f'{fname}: {{{", ".join(params_passed)}}}'
+        log.info(logstr)
```
Comment (eukreign) [inline lbry/lbry/extras/daemon/Daemon.py]: Does the default log formatting system not work for this?

Comment (osilkin98) [reply lbry/lbry/extras/daemon/Daemon.py]: No, only if either an error occurs, or a log is specified directly.

**Pull request merged**